src/NLog/Layouts/Typed/ValueTypeLayoutInfo.cs

```diff
@@ -173,6 +174,7 @@ public CultureInfo ValueParseCulture
         /// <summary>
         /// Render Result Value
         /// </summary>
+        /// <remarks>Inside a <see cref="Target"/>, <see cref="Target.RenderLogEvent"/> is preferred for performance reasons.</remarks>
```

Comment:
`RenderValue` on `ValueTypeLayoutInfo` is special, as it allows one to configure a Typed-Layout from the configuration-file and also has support for raw-values.

Reply:
reverted this change